index.html

```diff
@@ -5,5 +5,6 @@
 	
 	<body>
 		<p>This is a simple web page that I’m going to save in my Git repository</p>
+		<img src=”assets/img/placeholder.png” alt=”Placeholder Image” />
```

Comment:
I don't like the placeholder image, but as it's going to be replaced with branding, it'll do for now.